beets/ui/__init__.py

```diff
@@ -523,6 +530,13 @@ def _colorize(color, text):
         escape = COLOR_ESCAPE + "%im" % (DARK_COLORS[color] + 30)
     elif color in LIGHT_COLORS:
         escape = COLOR_ESCAPE + "%i;01m" % (LIGHT_COLORS[color] + 30)
+    elif color in MORE_COLORS:
+        # 38 == foreground color, but attributes as well as background
+        # color can be specified in the same string if desired, i.e:
+        # 38;5;196;1;3;4;5;7;48;5;220m
+        COLOR_ESCAPE = "\x1b[38;5;"
+        RESET_COLOR  = "\x1b[m"
```

Comment:
Shadowing a global variable with a local variable can make the flow of data pretty confusing… maybe we just need a new local variable that picks between the two strings?